app/Policies/AdminPolicy.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace App\Policies;
+
+use App\Models\Admin;
+use Illuminate\Auth\Access\HandlesAuthorization;
+use Illuminate\Auth\Access\Response;
+
+class AdminPolicy
+{
+    use HandlesAuthorization;
+
+    /**
+     * Determine whether the user can view any models.
+     *
+     * @param  \App\Models\Admin  $admin
+     * @return mixed
+     */
+    public function viewAny(Admin $admin)
+    {
+        //
+    }
+
+    /**
+     * Determine whether the user can view the model.
+     *
+     * @param  \App\Models\Admin  $admin
+     * @param  \App\Models\Admin  $admin
+     * @return mixed
+     */
+    // public function view(Admin $admin, Admin $admin)
+    // {
+    //     //
+    // }
+
+    /**
+     * Determine whether the user can create models.
+     *
+     * @param  \App\Models\Admin  $admin
+     * @return mixed
+     */
+    public function create(Admin $superAdmin)
+    {
+        return $superAdmin->is_super == 1
+        ? Response::allow()
+        : Response::deny('You do not authorized to create admin', 403);
```

Comment:
No need to return a message as it will be handled from the client-side. Just write the condition